content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -0,0 +1,223 @@
+calls its own `_getPowerboxOptions` method, which can conveniently be overridden
+to add our new commands.
+
+Since `mass_mailing` already overrides `_getPowerboxOptions`, let's just add our
+new command to it:
+
+.. code-block:: javascript
+
+   _getPowerboxOptions: function () {
+       const options = this._super();
+       // (existing code before the return statement)
+       options.categories.push({
+           name: _t('Documentation'),
+           priority: 300,
+       });
+       options.commands.push({
+           name: _t('Document'),
+           category: _t('Documentation'),
+           description: _t("Add this text to your mailing's documentation"),
+           fontawesome: 'fa-book',
+           priority: 1, // This is the only command in its category anyway.
+       });
+       return options;
+   }
+
+The :ref:`reference <reference>` for the definition of :ref:`commands <command>`
+and :ref:`categories <category>` can be found further on this page.
+
+.. note::
+   Think of wrapping the names and descriptions of your commands and categories
+   in the `_t` function for translation purposes.
+
+.. note::
+   To avoid out-of-control escalations, don't use random numbers for your
+   priorities: look at what other priorities already exist and choose your value
+   accordingly (like you would do for a z-index).
+
+Opening a custom Powerbox
+-------------------------
+
+Say we want to open the Powerbox to offer a couple of options, no more, and we
+don't need all the clutter of the other commands. This is possible by manually
+calling the `open` method of the Powerbox and passing it the commands and
+categories we want.
+
+Example: opening the Powerbox with a new set of commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We need the current instance of the Powerbox, which can be found in the current
+editor. In the `Wysiwyg class
+<{GITHUB_PATH}/addons/web_editor/static/src/js/wysiwyg/wysiwyg.js>`_, you will
+find it as `this.odooEditor.powerbox`.
+
+Now to open it with our custom "Document" command in a custom "Documentation"
+category:
+
+.. code-block:: javascript
+
+   this.odooEditor.powerbox.open(
+       [{
+           name: _t('Document'),
+           category: _t('Documentation'),
+           description: _t("Add this text to your mailing's documentation"),
+           fontawesome: 'fa-book',
+           priority: 1, // This is the only command in its category anyway.
+       }],
+       [{
+           name: _t('Documentation'),
+           priority: 300,
+       }]
+   );
+
+Filters
+-------
+
+There are three ways to filter commands:
+
+#. Via the `powerboxFilters` :ref:`Powerbox option <options>`.
+#. Via a given :ref:`command <command>`'s `isDisabled` entry.
+#. The user can filter commands by simply typing text after opening the
+   Powerbox. It will fuzzy-match that text with the names of the categories and
+   commands.
+
+Reference
+---------
+
+Category
+~~~~~~~~
+
+.. _category:
```

Comment:
Prefix your anchors with some context so that
- it's easier to understand to which part of the doc a `ref` links to,
- it doesn't collide with other anchors elsewhere in the doc.

```suggestion
.. _reference/frontend/editor/powerbox-category:
```
```suggestion
.. __reference/frontend/powerbox/category:
```